packages/pwa-kit-create-app/scripts/create-mobify-app.js

```diff
@@ -922,132 +924,115 @@ const runGenerator = async (
+                    answers.project.name
+                )
+                fs.writeFileSync(extensionMetaJsonPath, extensionMetaJsonContent)
+            }
         }
 
-        // Add selected Application Extensions to devDependencies and mobify object
-        const appExtensionDeps = selectedAppExtensions.reduce((acc, appExtensionName) => {
-            // Find the corresponding Application Extension details
-            const appExtensionDetails = context.availableAppExtensions.find(
-                (ext) => ext.value === appExtensionName
-            )
-            const version = appExtensionDetails ? appExtensionDetails.version : '1.0.0-dev'
+        processGeneratedExtension()
 
-            acc[appExtensionName] = extractAppExtensions
-                ? `file:./app/application-extensions/${appExtensionName}`
-                : version
-            return acc
-        }, {})
+        // Create the .npmignore file, excluding the typescript-minimal local dev project folder
+        createNpmIgnoreFile(outputDir, [`${LOCAL_DEV_PROJECT_DIR}/`])
 
-        updatePackageJson(p.resolve(outputDir, 'package.json'), {
-            name: getSlugifiedProjectName(context.answers.project.name || context.preset.id),
-            version: GENERATED_PROJECT_VERSION,
-            devDependencies: appExtensionDeps,
-            ...(selectedAppExtensions.length > 0 && {
-                mobify: {
-                    app: {
-                        extensions: selectedAppExtensions.map(
-                            (appExtensionName) => appExtensionName
-                        )
-                    }
+        npmInstall(devOutputDir, {
+            verbose,
+            projectName: localDevProjectContext.answers.project.name
+        })
+    } else {
+        processAppExtensions(selectedAppExtensions, extractAppExtensions, appExtensionsDir)
+    }
+
+    // Add selected Application Extensions to devDependencies and mobify object
+    const appExtensionDeps = selectedAppExtensions.reduce((acc, appExtensionName) => {
+        // Find the corresponding Application Extension details
+        const appExtensionDetails = context?.availableAppExtensions?.find(
+            (ext) => ext.value === appExtensionName
+        )
+        const version = appExtensionDetails ? appExtensionDetails.version : '1.0.0-dev'
+
+        acc[appExtensionName] = extractAppExtensions
+            ? `file:./app/application-extensions/${appExtensionName}`
+            : version
+        return acc
+    }, {})
+
+    updatePackageJson(p.resolve(outputDir, 'package.json'), {
+        name: getSlugifiedProjectName(context.answers.project.name || context.preset.id),
+        version: GENERATED_PROJECT_VERSION,
+        devDependencies: appExtensionDeps,
+        ...(selectedAppExtensions.length > 0 && {
+            mobify: {
+                app: {
+                    extensions: selectedAppExtensions.map((appExtensionName) => appExtensionName)
```

Comment:
Aside: this line does not come from this PR, but what is it supposed to do? It does not transform the list, except maybe for creating a copy of the list?

Reply:
Oh yeah.. I had that same comment when I was going over the code with @adamraya . We cam to the conclusion that it was probably just a result of refactoring some other code and not realizing that the map no longer did anything, it might have at some point in time. I'll fix it.